storage.js

```diff
@@ -0,0 +1,27 @@
+const contactForm = document.querySelector('#contact-form');
+
+function storeData(nameData, emailData, messageData) {
+  const formData = {
+    nameData,
+    emailData,
+    messageData,
+  };
+
+  localStorage.setItem('formData', JSON.stringify(formData));
+  // console.log(JSON.stringify(formData));
```

Comment:
- [ ] [OPTIONAL] You can remove this unnecessary commented code. avoid leaving comments like these it will make your code base a lot cleaner

Reply:
Addressed 🚀